x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/components/steps/components/experimental_datastream_settings.test.tsx

```diff
@@ -178,14 +178,91 @@ describe('ExperimentDatastreamSettings', () => {
           },
         },
       ]);
-      expect(experimentalDataFeatures).toEqual([
```

Comment:
This assertion wasn't doing anything, as we're just asserting on a variable we declared above